pentaho/pom.xml

```diff
@@ -9,26 +9,7 @@
   </parent>
   <artifactId>cpf-pentaho</artifactId>
   <version>8.1.0.0-SNAPSHOT</version>
-  <properties>
-    <dependency.jsr311-api.revision>1.1.1</dependency.jsr311-api.revision>
```

Comment:
Did all of this got moved into the root pom dependencyManagement? Any particular reason why?

Reply:
Because the dependency management section is a mechanism for centralizing dependency information. It's very helpful to have the information in a single place instead of spread through the several child POMs.
https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management